scripts/storage_apply.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+#
+# Copyright 2025 Red Hat Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+set -ex
+
+NODE=${1:-"crc"}
+OPERATION=${2:-"create"}
+
+oc delete -n openstack job "crc-storage-${NODE}" --ignore-not-found
+
+cat << EOF | oc apply -f -
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: crc-storage-${NODE}
+  namespace: openstack
```

Comment:
should we run this in the openstack-operators namespace? Or maybe even in its own "openstack-crc-storage" namespace to keep it isolated. I typically run crc_storage very early and won't always have this pre-created.